src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="c-context-menu" cdkMenu>
+    <div class="c-context-menu-header">Options</div>
+    <button *ngIf="rundown?.isActive" class="c-context-menu-item" cdkMenuItem (click)="deactivate()">Deactivate</button>
+    <button *ngIf="rundown?.isActive" class="c-context-menu-item" cdkMenuItem (click)="take()">Take</button>
+    <button *ngIf="!rundown?.isActive"class="c-context-menu-item" cdkMenuItem (click)="activate()">Activate</button>
```

Comment:
You could use templates for composing this.

Reply:
Templates have been put to use instead!